org.lflang/src/org/lflang/validation/LFValidatorImpl.xtend

```diff
@@ -1292,6 +1284,23 @@ class LFValidatorImpl extends AbstractLFValidator {
             }
         }
     }
+    
+    @Check(FAST)
+    def checkVarRef(VarRef varRef) {        
+        if (varRef.isInterleaved) {
+            if (this.target != Target.CPP && this.target != Target.C && this.target != Target.Python) {
+                error("This target does not support interleaved port referenced", Literals.VAR_REF__INTERLEAVED)    
+            }
+            if (varRef.container === null || varRef.container.widthSpec === null || 
+                (varRef.variable as Port).widthSpec === null
+            ) {
+                error("Interleaved can only be used for multiports contained within banks", Literals.VAR_REF__INTERLEAVED)            
```

Comment:
Shouldn't this also check that the port is a multiport?  Also, since `interleaved` is a keyword, I suggest not capitalizing here in the message.

Reply:
It does check if the port is a multiport in the second line of the if (line 1295).

Thank you for the thorough review by the way!